docs/Diagnostics/HealthChecker/RebootPending.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Reboot Pending
+parent: HealthChecker.ps1
+grand_parent: Diagnostics
+---
+
+It is best to reboot the server to address these issues. It may take some time after a reboot to have the keys automatically removed. However, if they don't remove automatically, follow these steps to address the issue for the keys that were provided to be a problem.
+
```

Comment:
Let's add a note here:

`Follow the steps in this section carefully. Serious problems might occur if you modify the registry incorrectly. Before you modify it, [back up the registry for restoration](https://support.microsoft.com/help/322756) in case problems occur.`